src/Engine/Animation/AnimationController.cpp

```diff
@@ -36,6 +36,20 @@ void AnimationController::Save(const std::string& path) {
         node->Save(&file);
     }
 
+    uint32_t numBools = boolMap.size();
+    file.write(reinterpret_cast<char*>(&numBools), sizeof(uint32_t));
+
+    for (BoolItem * b : boolMap) {
+        file.write(reinterpret_cast<char*>(&boolMap), sizeof(BoolItem));
```

Comment:
I assume it's `b` (which is already a pointer, btw) you want to write, not `&boolMap`?